### PR TITLE
Rel/latest

### PR DESCRIPTION
**Pull request by mmcshinsky-bitgo**

### CODE REVIEW

**Review by brandonblack — DISMISSED** [review]

**Review by sachushaji — COMMENTED** [review]
Unit test seems to be failing

File: modules/abstract-utxo/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitgo/abstract-utxo",
-  "version": "2.1.2",
+  "version": "2.2.1",
```
Comment (LeoKotschenreuther) [inline modules/abstract-utxo/package.json:3]: If we are going from version 2.1.2 and adding features (but no breaking changes), shouldn't the new version be 2.2.0?

**Review by LeoKotschenreuther — DISMISSED** [review]
I got confused about the versioning but this is not blocking.
Did we do a few releases since last changing our package.json files that could explain us skipping a few versions?

**Review by aarvay — COMMENTED** [review]
flushing from queue. required build step failing.

File: modules/abstract-utxo/package.json (hunk 1 above)
Comment (mmcshinsky-bitgo) [reply modules/abstract-utxo/package.json]: I'd have to look at all the commits. This PR has a couple stable releases in it, so 2.2.0 was probably farther back in the commit history

**Review by zahin-mohammad — DISMISSED** [review]

**Review by brandonblack — DISMISSED** [review]

**Review by sachushaji — DISMISSED** [review]

**Review by mukeshsp — DISMISSED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Review by margueriteblair — APPROVED** [review]

**Review by rlutzbitgo — APPROVED** [review]

**Review by brandonblack — APPROVED** [review]

**Pull request merged**